src/os/bluestore/BlueStore.h

```diff
@@ -305,33 +305,23 @@ class BlueStore : public ObjectStore,
     std::shared_ptr<int64_t> cache_age_bin;  ///< cache age bin
 
     boost::intrusive::list_member_hook<> lru_item;
-    boost::intrusive::list_member_hook<> state_item;
+    boost::intrusive::set_member_hook<>  set_item;
+
+    static std::atomic<uint64_t> total;
```

Comment:
this `total` should probably be under DEBUG macro or something

Reply:
let's keep it on unconditionally for now to make sure Buffers are not leaking. Using this in DEBUG build only might miss that given it's not that commonly used.

Reply:
I doubt so. But let's keep it through the full QA test and remove immediately before the commit